its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SonarQubeTestSuite.java

```diff
@@ -45,6 +45,7 @@ public class SonarQubeTestSuite {
     .addPlugin(FileLocation.of(TestUtils.getCustomRoslynPlugin().toFile()))
     .addPlugin(TestUtils.getMavenLocation("org.sonarsource.dotnet", "sonar-csharp-plugin", System.getProperty("sonar.csharpplugin.version", "DEV")))
     .addPlugin(TestUtils.getMavenLocation("org.sonarsource.dotnet", "sonar-vbnet-plugin", System.getProperty("sonar.vbnetplugin.version", "DEV")))
+    .addPlugin(TestUtils.getMavenLocation("org.sonarsource.javascript", "sonar-javascript-plugin", System.getProperty("sonar.javascriptplugin.version", "DEV")))
```

Comment:
Hardcode the JS plugin version here so it does not fail,  [like so.](https://github.com/SonarSource/sonar-dotnet/blob/master/its/src/test/java/com/sonar/it/shared/Tests.java#L54)